docs/sources/k6-studio/record-your-first-script.md

```diff
@@ -37,44 +37,49 @@ To create a test recording:
 
 1. Open the k6 Studio desktop application.
 1. Click **Record Flow**.
-1. Type "test-api.k6.io" under **Starting URL**.
+1. Type "quickpizza.grafana.com" under **Starting URL**.
 1. Click **Start recording**. You should see a Google Chrome window open after a few seconds, and the **Requests** header in the k6 Studio application.
 
    {{< figure src="/media/docs/k6-studio/tutorial/screenshot-k6-studio-tutorial-1-test-recorder-2.png" >}}
 
 1. In k6 Studio, click the edit button next to **Default group**, rename it to "Homepage", and press **Enter**.
-1. Next click **Create group**, name it “Go to Create Crocs”, and press **Enter**. Groups can help organize your test scripts so they're easier to edit and maintain in the future, and give you timing metrics for each group in your test script.
-1. In the browser window, under **Private APIs**, click on **/my/crocodiles** next to the **POST** method. You should see an authentication dialog box in the next screen.
-1. For username, enter "studio-user". For password, enter "password".
+1. Next click **Create group**, name it “Generate pizza”, and press **Enter**. Groups can help organize your test scripts so they're easier to edit and maintain in the future, and give you timing metrics for each group in your test script.
+1. In the browser window, press the **Pizza, Please!** button. You should see a generated recommendation.
+1. Under the recommendation, press the **Love it!** button, a message asking to login will appear.
 
-You'll be logged in to the test service, and this will complete the requests for the "Go to Create Crocs" group.
+This will complete the requests for the "Generate pizza" group.
 
 {{< figure src="/media/docs/k6-studio/tutorial/screenshot-k6-studio-tutorial-2-test-api-service-2.png" >}}
```

Comment:
This screenshot should probably be replaced, no?

Reply:
Yeah, I'm going to start working on updating all the screenshots. 🤓